src/server/metrics.rs

```diff
@@ -127,6 +127,7 @@ impl Metrics {
 
     pub fn incr_with_tags(&self, label: &str, tags: Option<Tags>) {
         if let Some(client) = self.client.as_ref() {
+            dbg!(&label, &tags);
```

Comment:
Kind of ok with keeping this, but that's probably the recent patches talking.

We should trim out extra `dbg!()`s